src/transform/shot_count_block/pattern.rs

```diff
@@ -192,6 +192,14 @@ impl<'ctx> ShotCountPatternMatchContext<'ctx> {
 
         Ok(pattern_context)
     }
+
+    /// Returns the parameters which do not have a constant value.
+    pub(crate) fn dynamic_parameters(&self) -> Vec<&FloatValue<'ctx>> {
```

Comment:
```suggestion
    pub(crate) fn get_dynamic_parameters(&self) -> Vec<&FloatValue<'ctx>> {
```

Just a style opinion given that this allocates a new vec and is not a property